pod/perldelta.pod

```diff
@@ -27,6 +27,15 @@ here, but most should go in the L</Performance Enhancements> section.
 
 [ List each enhancement as a =head2 entry ]
 
+=head2 Added C<newLISTOPn()>
```

Comment:
This isn't a core language feature "XXX New core language features go here.  Summarize user-visible core language
enhancements.".

Doesn't this belong under "=head1 Internal Changes", described as "XXX Changes which affect the interface available to C<XS> code go here."?

Reply:
Ahyes, oops. I put it in the wrong section.
force-pushed.